INSTALLATION_TROUBLESHOOTING.md

```diff
@@ -35,6 +35,19 @@
     2. When you run ElasticSearch, it seems that the process takes a very long time. 
 	To proceed the process, just let ElasticSearch to run
     on its own terminal, and proceed the building process by making another window of terminal
+	
+	3. If you run into an error on Docker Toolbox with Elastic Search stating an error message along the lines of:
+		"Waiting for elasticsearch:9200  .elasticsearch: forward host lookup failed: Unknown host"
+	   The cause could be the docker-machine's memory limits. you can inspect this with the command: docker-machine inspect machine-name
```

Comment:
There is a diagnostic step missing you forgot to mention where you looked at the logs and saw the error `There is insufficient memory for the Java Runtime Environment to continue`. That's the best clue that there is  memory limit issue.